### ISSUE TITLE
MaterialBox to utilize alt-tag for captions also?

### ISSUE THREAD

**Issue by tnmannen** [Dogfalo/materialize#3174]
Just a thought: For SEO and screen-reader purposes the alt-tag should be added on img elements.

If the alt-tag and the caption texts are the same why not provide an option to use alt-tag as fallback for data-caption?

Feels a bit silly to put:

``` html
<img src="src" alt="Description of the image yada yada" data-caption="description of the image yada yada">
```

One option could be to define data-caption="alt", in which case the caption would be picked up from the alt-tag?

``` html
<img src="src" alt="Description of the image yada yada" data-caption="alt">
```

**Comment by stefan-niedermann** [comment]
@tomscholz has this been implemented? I tried the proposed `data-caption="alt"`, but it just displayed "alt"...
